src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -35,19 +36,23 @@ public BookController(BookRepository bookRepository) {
     }
 
     /**
-     * This method returns all the books stored in the database
+     * This method returns all the books stored in the database through a filter
      *
      * @return {@link List<Book>}
      */
     @GetMapping
-    @ApiOperation(value = "return all book", response = Book[].class)
+    @ApiOperation(value = "Given a filter type and a param for filter, return all books", response = Book[].class)
     @ApiResponses(value = {
             @ApiResponse(code = 200, message = "Success"),
             @ApiResponse(code = 401, message = "Not Authorized"),
-            @ApiResponse(code = 403, message = "Access forbidden")
+            @ApiResponse(code = 403, message = "Access forbidden"),
+            @ApiResponse(code = 404, message = "Book not found"),
```

Comment:
Remember that this code can't happen, because the return value is a List, if there are no books then the list will be empty :)